### ISSUE TITLE
The messages in the dialog boxes are too brief

### ISSUE THREAD

**Issue by evaristor** [cbirdferrer/collatrix#4]
The messages in the dialog boxes (titles and names of fields) are too brief. On the other hand, the description in the README (and even the comments in the code) are satisfactorily detailed. I suggest adding text boxes to the GUI with more detailed information.

---

- [Review for the Journal of Open Source Software](https://github.com/openjournals/joss-reviews/issues/2328)

**Comment by cbirdferrer** [comment]
Thanks for this suggestion! We feel that adding more to the dialog box titles would be a little too wordy, which is why we added detail to the README and matched the order of the inputs to the order of the inputs in the README. That being said, I agree that the dialog could be better. To improve the link between the README and the dialog boxes, I've added numbers to the box titles that match those of the README. For example "Input 2. Do you want to use the safety?".

**Comment by evaristor** [comment]
That looks good!
:+1: :100:
